### ISSUE TITLE
Wrong `role` assigned to `select` component

### ISSUE THREAD

**Issue by uraaaa24** [yamada-ui/yamada-ui#3167]
### Which package has an issue?

@yamada-ui/react

### Package Version

1.5.4

### Description

`comobox` is assigned to `combobox` when `listbox` should be assigned.

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

Chrome

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iPadOS
- [ ] iOS
- [ ] Android

### Do you know how to fix the issue?

Yes

### Are you willing to participate in fixing this issue and create a pull request with the fix?

Yes

**Comment by hirotomoyamada** [comment]
@uraaaa24 

Hi, The `Select` component is correct with `role="combobox"`.

However, there are a few things that need to be corrected. I will correct it on my end.
